scripts/commands/quest.lua

```diff
@@ -0,0 +1,152 @@
+-----------------------------------
+-- func: addallspells
```

Comment:
Looks like the info at the top of the file needs to be updated for this command

Reply:
ahh, true. my secret. i like to take a page edit what i need instead of create all from scratch